src/quicktype-core/language/CPlusPlus.ts

```diff
@@ -649,7 +826,7 @@ export class CPlusPlusRenderer extends ConvenienceRenderer {
 
     protected emitDeclaration(decl: Declaration): void {
         if (decl.kind === "forward") {
-            this.emitLine("struct ", this.nameForNamedType(decl.type), ";");
+            this.emitLine("class ", this.nameForNamedType(decl.type), ";");
```

Comment:
Shouldn't this be `struct` or `class`, depending on the options?

Reply:
F@ckit, thanks!